src/style.css

```
@@ -0,0 +1,66 @@
+html {
```

```suggestion
@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");


html {
```

It's should fix the test. Try